src/Maestro/Maestro.Web/Controllers/AzDevController.cs

```diff
@@ -33,12 +33,13 @@ public AzDevController(IAzureDevOpsTokenProvider tokenProvider)
         }
 
         [HttpGet("build/status/{account}/{project}/{definitionId}/{*branch}")]
-        public async Task<IActionResult> GetBuildStatus(string account, string project, int definitionId, string branch, int count)
+        public async Task<IActionResult> GetBuildStatus(string account, string project, int definitionId, string branch, int count, string status)
         {
             string token = await TokenProvider.GetTokenForAccount(account);
+            
             return await HttpContext.ProxyRequestAsync(
                 s_lazyClient.Value,
-                $"https://dev.azure.com/{account}/{project}/_apis/build/builds?api-version=5.0&definitions={definitionId}&branchName={branch}&statusFilter=completed&$top={count}",
+                $"https://dev.azure.com/{account}/{project}/_apis/build/builds?api-version=5.0&definitions={definitionId}&branchName={branch}&statusFilter={status}&$top={count}",
```

Comment:
We are using `$top` here but there is no `queryOrder` parameter specified. I'm not sure if AzDO API has some predetermined order or is it random. In any case I think it would be safer to add explicit ordering here so that we always get the elements we want.

Maybe it should even be exposed as parameter because probably in different places in the code we want to sort results in different way but if it's not necessary at the moment then just hardcoding it would be enough.

Reply:
I agree, however there wasn't _order_ specified in former version of code as well. As the default sort order is not specified in AzDO API docs, it could be specific sort order (for example `finished ?? started`) which other logic in our solution can relay on. For above reasons I recommend to left it as is.